index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
```

Comment:
If you open up `index.html` in the browser and open the web console, you'll see a few errors are thrown. The error messages will helpfully point out which line of code the error is coming from 🙂